Parse/src/main/java/com/parse/ParseRESTAnalyticsCommand.java

```diff
@@ -29,12 +29,12 @@
   private static final String PATH = "events/%s";
 
   private static final String KEY_AT = "at";
-  private static final String KEY_PUSH_HASH = "push_hash";
+  private static final String KEY_PUSH_ID = "push_id";
```

Comment:
The key is actually `parsePushId`. The reason why we chose that is because that key was already being used to send the push id to the iOS SDK.